src/simulation/simulation_store_params.jl

```diff
@@ -3,7 +3,7 @@ struct SimulationStoreParams
     step_resolution::Dates.Period
     num_steps::Int
     # The key order is the problem execution order.
-    models::OrderedDict{Symbol, ModelStoreParams}
+    problems::OrderedDict{Symbol, ModelStoreParams}
```

Comment:
If the type is ModelStoreParams, isn't `models` a better term?

Reply:
I switched to problems since the key is the problem name.